### PR TITLE
HOOK-3702 Add verbose output for isExistingStorageAccountAndContainer

### PR DESCRIPTION
**Pull request by dan-leanix**

### CODE REVIEW

**Review by perbernhardt — APPROVED** [review]
❤️  THANKS A LOT!

**Comment by konstantintieber** [comment]
This is awesome. Thanks so much!

File: azcopy-command.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ class AzcopyCommand {
             this.source,
             this.destination,
             ...this._getArgumentList()
-        ]);
+        ], {silent: true});
```
Comment (konstantintieber) [inline azcopy-command.js:91]: Why exactly does this one need to be silent?  I liked to see the upload progress in percentage while the azcopy command was running.

Did the output of this command also expose the sasToken?

Comment (dan-leanix) [reply azcopy-command.js:91]: @konstantintieber Yes, as it is part of the generated `this.destination`.

Comment (konstantintieber) [reply azcopy-command.js:91]: I see. I'll try to remember to look into the `@actions/core` [setSecret](https://github.com/actions/toolkit/blob/27f76dfe1afb2b7e5e679cd8e97192d34d8320e6/packages/core/src/core.ts#L101-L103) function, which will mask the given string in any logs that our action outputs.

Link to docs: https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#masking-a-value-in-log

**Pull request merged**